lib/spark_api/models/listing.rb

```diff
@@ -235,6 +235,10 @@ def respond_to?(method_symbol, include_all=false)
         end
       end
 
+      def batch_photo_delete(photoIds, args={})
```

Comment:
Good point, @ryanhertz. A bit more reading on my part confirms this as a common way of thinking about it.

Do you have any thoughts on reconciling the two methods? Using a bang here makes things more consistent, avoiding the "arbitrary subset" issue, but also moves us further away from being idiomatic.

Another thing that may affect the direction here -- what happens if the delete call fails? If it returns an error, the bang may be justified on those grounds.